tests/unittest_brain_qt.py

```diff
@@ -0,0 +1,36 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+from importlib.util import find_spec
+
+import pytest
+
+from astroid import extract_node
+from astroid.bases import UnboundMethod
+from astroid.manager import AstroidManager
+from astroid.nodes import FunctionDef
+
+HAS_PYQT6 = find_spec("PyQt6")
+
+
+@pytest.mark.skipif(HAS_PYQT6 is None, reason="This test requires the PyQt6 library.")
```

Comment:
Maybe this test is platform specific? It passes on my mac.

Reply:
(I don't know how to verify the C extensions were installed on linux.)

Reply:
Do you have ``PyQt6`` installed ? Maybe it was skipped because of line 34 ?

Reply:
Yes the test passes for me on a Mac with PyQt6 installed and skips properly when it's not installed. I added a `pip list` command to another branch and opened a PR against my own fork, and verified that `PyQt6` is still installed on CI. Somehow the inference is still `Uninferable`. I'm wondering if that's because on linux `pip install` doesn't necessary install the c bindings. I looked at the docs. Something about `qmake must be on your PATH`?

Reply:
>  I'm wondering if that's because on linux pip install doesn't necessary install the c bindings.

Yes, we need the c-binding installed for the system before launching pip. But we should probably separate tests with pyqt installed vs our standard tests. We don't want astroid to unexpectedly break if pyqt6 is not installed. Right now we're skipping some tests if numpy/pandas/pyqt is not installed, but we might want to create a mark and specific runner with the required dependencies to cover all the tests that requires a dependency or another. I

Reply:
Added a test to the "release tests" workflow that when manually triggered on my fork, passed!